test/functional/apps/discover/group1/_sidebar.ts

```diff
@@ -424,11 +424,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.discover.waitUntilSidebarHasLoaded();
 
         expect(await PageObjects.discover.getSidebarAriaDescription()).to.be(
-          '0 available fields. 0 meta fields.'
+          '0 available fields. 0 empty fields. 0 meta fields.'
         );
+        // ask team how this should be handled
+        /*
         await testSubjects.existOrFail(
           `${PageObjects.discover.getSidebarSectionSelector('available')}-fetchWarning`
         );
+        */
```

Comment:
that's a good question, given we no longer treat missing indices as error, there's also no reason to show a warning here IMO